.github/workflows/equivalence-test-update.yml

```
@@ -18,14 +21,14 @@ jobs:
       - name: target_branch
         id: target_branch
         run: |
-          merged = ${{ github.event.pull_request.merged }}
-          target_branch = ${{ github.event.pull_request.base.ref }}
+          merged="${{ github.event.pull_request.merged }}"
+          target_branch="${{ github.event.pull_request.base.ref }}"
```

```suggestion
          target_branch='${{ github.event.pull_request.base.ref }}'
```
I think this is slightly safer as it disallows interpolation, so that in case the ref contained some special characters, like `$()` then it won't have any special meaning if it's enclosed in `'` rather than `"`. The only special character then becomes just `'`.

Hopefully GitHub disallows branch names with special characters anyway but better safe than sorry.